fixtures/nextest-tests/tests/basic.rs

```diff
@@ -319,6 +319,16 @@ fn test_slow_timeout_subprocess() {
     cmd.output().unwrap();
 }
 
+#[test]
+#[ignore]
+fn test_flaky_slow_timeout_mod_3() {
+    let nextest_attempt = nextest_attempt();
+    if nextest_attempt % 3 != 0 {
+        panic!("Failed because attempt {} % 3 != 0", nextest_attempt)
+    }
+    std::thread::sleep(std::time::Duration::from_secs(4));
```

Comment:
Where does this number come from? See the comments in the lines above, could you add a similar explanation?